engine/apps/auth_token/auth.py

```diff
@@ -133,6 +134,14 @@ def _get_user(request: Request, organization: Organization) -> User:
         except KeyError:
             user_id = context["UserID"]
 
+        if context.get("IsServiceAccount", False):
```

Comment:
This just reminded me we need to clean-up the GrafanaHeadersMixin in mixins.py at some point whenever we remove the old status endpoint.  This way of doing it is better in that it is more tolerant of missing/different fields when dealing with plugin rollout difference between objects.